ci-operator/step-registry/windows/e2e/operator/test/mirror-images/windows-e2e-operator-test-mirror-images-commands.sh

```diff
@@ -55,14 +55,13 @@ do
     source_image=$(echo "$image" | cut -d'=' -f1)
     mirror_placeholder=$(echo "$image" | cut -d'=' -f2)
 
-    icsp_content+="  - mirrors:\n"
-    icsp_content+="    - $mirror_placeholder\n"
-    icsp_content+="    source: $source_image\n"
+    itms_content+="  - source: $source_image\n"
+    itms_content+="  destination: $mirror_placeholder\n"
```

Comment:
please test this locally as I showed on slack. this does not generate a valid ITMS